cactusbot/cached/caching.py

```diff
@@ -1,18 +1,32 @@
+import json
+import time
+
+
 class CacheUtils(object):
 
     # You have found MysticalMage's code, turn back now, no good lies ahead
-    def in_cache_followers(user):
-        """ Returns if user is in the cache. """
-        follower_cache = open("caches/CacheFollowers.txt", "r")
-        for line in follower_cache:
-            if user in line:
-                follower_cache.close()
-                return True
-        follower_cache.close()
+    def __init__(self, cacheName):
+        super().__init__()
+
+        self.cacheName = cacheName
+
+    def in_cache(self, user):
+        """ Returns if user is in the cache, and if so, the time in secs """
```

Comment:
Seems to just return `True` or `False` based on whether or not the user is in the cache. Should either fix that functionality or change the docstring to reflect what it does.